Firestore/core/src/firebase/firestore/local/memory_mutation_queue.h

```diff
@@ -52,11 +52,7 @@ class MemoryMutationQueue {
 
   void Start();
 
-  bool is_empty() {
-    // If the queue has any entries at all, the first entry must not be a
-    // tombstone (otherwise it would have been removed already).
-    return queue_.empty();
-  }
+  bool IsEmpty();
```

Comment:
I have no objection to the changes you've made to MemoryMutationQueue, but I'm not sure they were necessary either. These seem like reasonable accessors/mutators.

Update: Oh, I see:

> Note that I changed the name of a couple methods in the memory version as well, since they will need to line up in the next PR when the actual interface is ported.

sgtm.